src/oidcprovider/adapters/grantAdapter.ts

```diff
@@ -19,31 +30,45 @@ export const GrantPayload = t.type({
     scope: t.string,
   }),
 });
+type GrantPayload = t.TypeOf<typeof GrantPayload>;
 
 export const toAdapterPayload = (input: Grant): oidc.AdapterPayload => ({
   accountId: input.accountId,
   clientId: input.clientId,
-  exp: input.expireAt,
-  iat: input.issuedAt,
+  exp: toNumericDate(input.expireAt),
+  iat: toNumericDate(input.issuedAt),
```

Comment:
exp and iat are second based?

Reply:
Yep. They are in `NumericDate` format, like the field in JWT [RFC-7519](https://datatracker.ietf.org/doc/html/rfc7519)